content/resources/beginners.md

```diff
@@ -39,10 +39,36 @@ library("swirl")
   + [R for Social Scientists](https://datacarpentry.org/r-socialsci/)
   + [R for Ecologists](https://datacarpentry.org/R-ecology-lesson/)
 
+## Presentations/videos
+- [RMarkdown](https://slides.djnavarro.net/starting-rmarkdown/#1) by Danielle Navarro
+
+- [Data validation](https://allisonhorst.shinyapps.io/missingexplorer/) by Allison Horst
+
+- [Data visualisation](https://www.youtube.com/c/DanielleNavarro77) by Danielle Navarro
+
+- [Intro to R](https://nhs-r-community.github.io/intro_r/) by Andrew Jones and Ozayr Mohammed
+
+- [Introduction to GIS in R](https://annakrystalli.me/intro-r-gis/index.html) by Anna Krystalli
+
+
+## Tutorials
```

Comment:
See if you can find an emoji to go with Tutorials (to keep the style consistent).